### ISSUE TITLE
Panic on 'for .. of' loop in async function

### ISSUE THREAD

**Issue by cauthmann** [swc-project/swc#600]
Thanks for your quick work on 598. I tried to run swc over our test-suite to get you some real-world testcases, but got consistent panics on several files. I reduced it to a minimal testcase which consistently panics, both with the official binaries and with `npm_config_build_from_source=1`, on x64 linux.

The minimal testcase is this:
```javascript
async function foo() {
	for (let a of b) {
	}
}
```
Removing the async fixes it, removing the loop (or using a different kind of loop) fixes it.

` npx swc --no-swcrc test.js`
```
fatal runtime error: failed to initiate panic, error 5
```
No way to get a panic message or backtrace, unfortunately. I wish there was a cli binary that didn't go through node..

**Comment by kdy1** [comment]
I've published a new version

**Comment by cauthmann** [comment]
Confirmed fixed in 1.1.14, thank you!
